teams.py

```diff
@@ -25,9 +25,9 @@ def hero_generator(self):
         generates particular hero
         :return:
         '''
-        types_of_characters = [character.Sorceress, character.Warrior, character.Support]
-        chosen_char = types_of_characters[random.randint(0, 2)]
-        if chosen_char is character.Sorceress:
+        types_of_characters = [characters.Sorceress, characters.Warrior, characters.Support, characters.Voodoo]
+        chosen_char = types_of_characters[random.randint(0, len(types_of_characters) - 1)]
+        if chosen_char in [characters.Sorceress, characters.Voodoo]:
```

Comment:
Skoro klasy maja i tak na sztywno wpisane HP tutaj... to może lepiej to przerzucić do definicji klas?

Reply:
zrobie jak bede mial sile, poki co jak wracam do pokoju, to jedyne co mi sie chce to spac, ale dzieki za komenty